Entities/Round.cpp

```diff
@@ -0,0 +1,132 @@
+#include "Round.h"
+#include "PresetMan.h"
+#include "MOPixel.h"
+
+namespace RTE {
+
+	ConcreteClassInfo(Round, Entity, 0)
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	void Round::Clear() {
+		m_ParticleCount = 0;
+		m_Particle = 0;
+		m_Shell = 0;
+		m_FireVel = 0;
+		m_ShellVel = 0;
+		m_Separation = 0;
+		m_AILifeTime = 0;
+		m_AIFireVel = -1;
+		m_AIPenetration = -1;
+		m_FireSound.Reset();
+	}
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	int Round::Create() {
+		if (Entity::Create() < 0) {
+			return -1;
+		}
+
+		// Check for AI overrides and if not defined use original preset values
```

Comment:
It doesn't hurt.

Reply:
It's just a pointless comment imo since the code is pretty clear and straightforward; I don't really think it adds anything that the code, which basically says `if (there's no ai value) { calculate and set ai value; }`, doesn't already say.